src/core/issues-notes.js

```diff
@@ -241,10 +241,16 @@ function isLight(rgb) {
  */
 function createLabelsGroup(labels, title, repoURL) {
   const labelsGroup = labels.map(label => createLabel(label, repoURL));
+  let labelNames = labels.map(label => label.name);
+  labelNames = joinAnd(labelNames);
   if (labelsGroup.length) {
     labelsGroup.unshift(document.createTextNode(" "));
   }
-  return hyperHTML`<span style='text-transform: none'>: ${title}${labelsGroup}</span>`;
+  const ariaLabel = `This issue is labelled as ${labelNames}`;
```

Comment:
I think we need a full stop here, right? 
```suggestion
  const ariaLabel = `This issue is labelled as ${labelNames}.`;
```